### ISSUE TITLE
🏗✨ Omakase Ruby styling for Rails

### ISSUE THREAD

**Issue by DerekNonGeneric** [OpenINF/openinf-jekyll-asset-path-plugin#3]
this one is more a 2fer to hopefully help us blend into the ruby community a little bit better (we are still newcomers and unsure how welcome we may be; have yet to prove ourselves, but they seem nice)

***TL;DR:***

> [We] will be able to have desired ruby linting from RoR project over in separate repo [this one now, actually]
> &mdash; https://github.com/OpenINF/openinf.github.io/pull/1131#discussion_r1502630984

***sauce:***

<blockquote class="twitter-tweet"><p lang="en" dir="ltr">Rails 8 will ship with a default Rubocop linter for the first time. The styles will come from this omakase menu. Substitutions will be easily achieved (and even encouraged). Find the Ruby flavor that tickles your fancy! <a href="https://t.co/4UW0Eberpl">

![nVnWf2Va](https://github.com/OpenINF/openinf-jekyll-asset-path-plugin/assets/17770407/b6a76920-45c0-4725-8135-c7bbb6493155)

</a></p>&mdash; DHH (@dhh) <a href="https://twitter.com/dhh/status/1740754874474868959?ref_src=twsrc%5Etfw">December 29, 2023</a></blockquote>

----

***TL;DR:***

> (※ OpenINF CQ Checks now to be re-impl'd here as well)
> &mdash; #1

***sauce:***

<blockquote class="twitter-tweet"><p lang="en" dir="ltr">We are also doubling down on setting good habits for new applications with a default GitHub CI configuration that runs brakeman, rubocop, and tests automatically. <a href="https://t.co/WTCSNQfKDp">

![zzIoD-rw](https://github.com/OpenINF/openinf-jekyll-asset-path-plugin/assets/17770407/4da6cd86-dc0d-4d29-a36a-cbaf2438b451)

</a></p>&mdash; DHH (@dhh) <a href="https://twitter.com/dhh/status/1741838676487737763?ref_src=twsrc%5Etfw">January 1, 2024</a></blockquote>

**Comment by DerekNonGeneric** [comment]
i'll tackle this soon; currently jammed 'til ~5pm est (classtime atm)

**Comment by DerekNonGeneric** [comment]
well, i seem to have gotten diverted &mdash; going for dinner now &mdash; 11PM EST ill be ready to tackle these two issues around that time

**Comment by DerekNonGeneric** [comment]
running ~1hr behind schedule (catching up on email, as dumb as it sounds)

**Comment by DerekNonGeneric** [comment]
first time mucking around w/ rubocop?
having a hard time getting in the mood?
remedy:

👂 https://open.spotify.com/track/4w2Furap0ZzaiK4J14E9sR

not sure why tho

**Comment by DerekNonGeneric** [comment]
(i do tend to ruthlessly overmoderate myself into oblivion,, huh; i am still testing how GitHub comments can be totally abusive to contributors); pls do not do me like that ♥️

**Comment by DerekNonGeneric** [comment]
we actually have to first tackle #2 (no choice)
